src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -818,41 +845,6 @@ private static PostgresSqlDb getPostgresDB(@Named(Constants.POSTGRES_DB_URL) fin
         return postgresDB;
     }
 
-    /**
-     * Gets instance of user badge database liason manager
-     *
-     * @param postgresDB database
-     * @return concrete instance of IUserBadgePersistenceManager
-     */
-    @Provides
-    @Singleton
-    @Inject
-    private static IUserBadgePersistenceManager getUserBadgePersistenceManager(final PostgresSqlDb postgresDB) {
-
-        if (null == userBadgePersitenceManager) {
-            userBadgePersitenceManager = new PgUserBadgePersistenceManager(postgresDB);
-        }
-        return userBadgePersitenceManager;
-    }
-
-
-    /**
-     * Gets instance of the user streaks manager
-     *
-     * @param postgresDB database
-     * @return concrete instance of IUserStreaksManager
-     */
-    @Provides
-    @Singleton
-    @Inject
-    private static IUserStreaksManager getUserStreaksManager(final PostgresSqlDb postgresDB) {
-
-        if (null == userStreaksManager) {
-            userStreaksManager = new PgUserStreakManager(postgresDB);
-        }
-        return userStreaksManager;
-    }
-
```

Comment:
These did not need to be singletons at all, so I moved them to normal Guice `.bind()`'s above.